sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -211,8 +218,8 @@ class EventGridEvent(InternalEventGridEvent, EventMixin):
         'data_version': {'key': 'dataVersion', 'type': 'str'},
     }
 
-    def __init__(self, data, subject, event_type, data_version, **kwargs):
-        # type: (object, str, str, str, Any) -> None
+    def __init__(self, subject, event_type, data, data_version, **kwargs):
```

Comment:
I guess this is also a breaking change worth to note in the changelog as the position of positional argument changes.

Reply:
well, this is just reverted back to the original form :( 
i had a pr before that changed this, but we decided not to anyway